nlpiper/transformers/embeddings.py

```diff
@@ -12,23 +15,50 @@
 from nlpiper.logger import log
 
 __all__ = [
-    "GensimEmbeddings"
+    "GensimEmbeddings",
+    "TorchTextEmbeddings"
 ]
 
 
 class GensimEmbeddings(BaseTransformer):
-    """Gensim Embedding extraction."""
+    """Gensim Embedding extraction.
 
-    def __init__(self, keyed_vectors: Any, apply_doc: str = 'mean'):
-        """Stem tokens.
+    Callable arguments:
+
+    Args:
+            keyed_vectors (Any): Gensim model based on keyedVectors,
+                see more in: https://radimrehurek.com/gensim/models/keyedvectors.html
+            apply_doc (str): Mode to calculate the embeddings vector for the document,
+                which could be `"mean"` or `"sum"` of the tokens.
```

Comment:
These are not the callable arguments, we are using this section to document the `__call__` method.

```suggestion
            doc (Document): document to be cleaned.
            inplace (bool, default False): if False will return a new document object,
                                 otherwise will change the object passed as parameter and return None.
```
Don't accept the suggetion because I think the identation is not right 😅